CPP/SortingAlgrothims/Sort.cpp

```diff
@@ -72,6 +72,41 @@ void SortingAlgrothim<T>::InsertionSort(T arr[], uint32_t Size, SortingType_t So
         arr[j+1] = key;
     }
 }
+template<typename T>
+void SortingAlgrothim<T>::CountingSort(T arr[], uint32_t size,uint32_t Range, SortingType_t SortingType )
```

Comment:
We want the same interface for all sorting functions which takes `array` `size` and `sortingType` . You  can evaluate the `range` by getting the maximum and minimum element in the array